samples/batch/transcription-enabled-storage/Connector/Serializable/TranscriptionResult/RecognizedPhrase.cs

```diff
@@ -0,0 +1,49 @@
+﻿// <copyright file="RecognizedPhrase.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class RecognizedPhrase
```

Comment:
is this the SegementResult? I would keep use the same name as API returns to avoid confusing customers.

Reply:
Using the names of the V3 API